indico/modules/events/papers/templates/_contributions.html

```diff
@@ -47,15 +47,24 @@ <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
                 <h2>{% trans %}Editing{% endtrans %}</h2>
                 </div>
             </div>
-            {% for editable in contribution.editables -%}
-                <div>
-                    <a href="{{ url_for('event_editing.editable', editable) }}">
-                        {%- trans type=editable.type.name %}
-                            Go to timeline ({{ type }})
-                        {% endtrans -%}
-                    </a>
+            {% if contribution.editables %}
```

Comment:
Shouldn't we also take `contribution.allowed_types_for_editable` into account here, e.g. using a new property on Contribution that returns the editable that both exist but are also for enabled types?

I guess it's an edge case but it's strange to show editable types there even if they've been disabled later..